### DISCUSSION TITLE
Could not resolve dependency graphql@"15.3.0"

### DISCUSSION THREAD

**Discussion by tigerblue77** [requarks/wiki#5251] [Error / Bug Report] [answered]
Hello,

I got this error while updating npm packages :

![image](https://user-images.githubusercontent.com/37409593/166213853-77c239b2-a3ab-41a8-9cbc-3415f7f09505.png)

I'm using :
- Debian 11.3
- NodeJS v16.15.0
- WikiJS 2.5.279

Seems to be the same error than :
- https://github.com/requarks/wiki/discussions/4939
- https://github.com/requarks/wiki/discussions/4993

**Comment by daddyfix** [comment]
After upgrade I am having the same error. Did anyone solve this?

**Reply by daddyfix** [reply]
[SOLVED]
bad usb drive on raspberry pi 4. 
changed usb and loaded Ubuntu Server 20.4.1 LTS on RPi4
Now working well. no errors

**Comment by lovelock** [comment]
Maybe it's the node version issue. I tried nodejs v14 and it goes well. unfortunately cpu-features package has some issue with v14 on M1 chip macOS, so 
1. linux x86_64 works with nodejs v14
2. macOS with M1 gets cpu-features related errors with v14 and the above errors with v16

**Comment by simone80an** [comment]
I have the same problem after upgrade node.js from version 16 to version 20 (specifically v20.4.0) and npm to version 9 (specifically v9.9.3) on a Windows Server 2019 VM and solved using:

`npm install --legacy-peer-deps`

I run latest stable version of Wiki.js (2.5.303), as i see from Requirements page node.js 20 is supported since v2.5.300, so why facing this dependency problem?

**Comment by NGPixel** [comment, marked as answer]
**DO NOT** manually upgrade npm dependencies. The installation package already comes with the required dependencies pinned to the correct version.
